prov/opx/include/rdma/opx/fi_opx_reliability.h

```diff
@@ -134,7 +140,10 @@ struct fi_opx_reliability_service {
 	/* == CACHE LINE == */
 		RbtHandle				flow;		/*  1 qw  =   8 bytes */
 		uint64_t				ping_start_key;
-		uint64_t				unused;
+		uint16_t max_uncongested_pings;
+		uint16_t max_congested_pings;
+		uint8_t congested_flag;
+		uint8_t unused_padding2[3];
```

Comment:
Please try to keep consistent alignment/spacing with the rest of the structure.

Reply:
I'll fix this in a separate commit.